java/semantickernel-core/src/main/java/com/microsoft/semantickernel/KernelDefault.java

```diff
@@ -29,29 +35,34 @@
 public class KernelDefault implements Kernel {
```

Comment:
Should be named `DefaultKernel`?

Elsewhere in the Java code class have been named `Default<Foo> implements <Foo>`